Tests/MapboxNavigationTests/VanishingRouteLineTests.swift

```diff
@@ -292,4 +292,35 @@ class VanishingRouteLineTests: TestCase {
         let currentLineGradientStops = navigationMapView.routeLineGradient(congestionFeatures, fractionTraveled: 0.0)
         XCTAssertEqual(currentLineGradientStops[0.0], navigationMapView.trafficUnknownColor, "Failed to use trafficUnknownColor for route line when no congestion level found.")
     }
+    func testFindDistanceToNearestPointOnCurrentLine() {
+        // https://github.com/mapbox/mapbox-navigation-android/blob/0478c43781fdf7489f10f22c0055fadf181970f6/libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtilsTest.kt#L814
+        
+        let route = getMultilegRoute()
+        let coordinate = route.shape!.coordinates[15]
```

Comment:
I think that in such cases it's better to add assert check whether coordinate is really available.

Reply:
Thanks for the suggestion, I'll add check for the coordinate.